tests/IBTK/samraidatacache_01_2d.mpirun=1.output

```diff
@@ -0,0 +1,165 @@
+number of patch data indices before registering double precision vars:2
+number of patch data indices before registering single precision vars:11
+number of patch data indices before registering integer vars:20
+cloning pass 0
+number of patch data indices before cloning double precision vars:29
+number of patch data indices before cloning single precision vars:38
+number of patch data indices before cloning integer vars:47
+number of patch data indices after cloning all vars:56
+cloning pass 1
+number of patch data indices before cloning double precision vars:56
+number of patch data indices before cloning single precision vars:56
+number of patch data indices before cloning integer vars:56
+number of patch data indices after cloning all vars:56
+number of patch data indices after removing all cloned vars:56
+number of patch data indices after the cache goes out of scope:56
```

Comment:
Shouldn't this value go down? Maybe I don't understand this.

Reply:
No — they are cached, so they can be reused elsewhere.

Reply:
The max number of indices means the max number encountered so far in the program. These are stored in a simple vector (presumably to make lookup fast), this number is basically the size of that array.

When you remove a patch data index, it just sets the corresponding slot in the vector to null.